src/mlpack/methods/random_forest/CMakeFiles/3.16.3/CMakeCXXCompiler.cmake

```diff
@@ -0,0 +1,88 @@
+set(CMAKE_CXX_COMPILER "/usr/bin/c++")
```

Comment:
This one should go also.  (I think there are a lot of these files, so, I'll probably duplicate this comment a lot---sorry about that.)